src/components/course/data/service.js

```diff
@@ -85,41 +90,47 @@ export default class CourseService {
     return this.cachedAuthenticatedHttpClient.get(url);
   }
 
-  async fetchEnterpriseUserSubsidy() {
-    // TODO: this will likely be expanded to support codes/offers, by appending to
-    // the below array to provide a function that makes the relevant API call(s)
-    // for the specified subsidy type.
+  async fetchEnterpriseUserSubsidy({ offers, catalogList }) {
+    // TODO: the license subsidy is fetched from backend, but the offer subsidy currently is being
+    //       used from the passed in arguments (fetched already by UserSubsidy.jsx).
+    //       Need to reconcile api to make it consistent for offers vs subsidies
```

Comment:
tiniest of nits: `...consistent for all subsidy types, e.g. license, offers` as they are all considered subsidies here.